sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -141,15 +141,20 @@ case class RowDataSourceScanExec(
       handledFilters
     }
 
-    val topNOrLimitInfo =
+    val limitOrOffsetInfo =
       if (pushedDownOperators.limit.isDefined && pushedDownOperators.sortValues.nonEmpty) {
-        val pushedTopN =
+        val topNStr =
           s"ORDER BY ${seqToString(pushedDownOperators.sortValues.map(_.describe()))}" +
           s" LIMIT ${pushedDownOperators.limit.get}"
-        Some("PushedTopN" -> pushedTopN)
-    } else {
-      pushedDownOperators.limit.map(value => "PushedLimit" -> s"LIMIT $value")
-    }
+        if (pushedDownOperators.offset.isDefined) {
+          Map("PushedPaging" -> s"$topNStr OFFSET ${pushedDownOperators.offset.get}")
```

Comment:
do we need to combine them into one metrics? I think we can have `PushedTopN` and `PushedOffset` together in the metrics.